src/coreComponents/python/modules/pygeosx_tools_package/pygeosx_tools/wrapper.py

```diff
@@ -0,0 +1,407 @@
+
+import sys
+import numpy as np
+from mpi4py import MPI
+import matplotlib.pyplot as plt
+import pylvarray
+import pygeosx
+
+
+# Get the MPI rank
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+
+
+def get_wrapper(problem, target_key, write_flag=False):
+    """
+    Get a local copy of a wrapper as a numpy ndarray
+
+    Args:
+        filename (str): Catalog file name
+        problem (pygeosx.Group): GEOSX problem handle
+        target_key (str): Key for the target wrapper
+        write_flag (bool): Sets write mode (default=False)
+
+    Returns:
+        np.ndarray: The wrapper as a numpy ndarray
+    """
+    local_values = problem.get_wrapper(target_key).value()
+    if write_flag:
+        local_values.set_access_level(pylvarray.MODIFIABLE,
+                                      pylvarray.CPU)
+    else:
+        local_values.set_access_level(pylvarray.CPU)
+
+    if not isinstance(local_values, np.ndarray):
+        local_values = local_values.to_numpy()
+    return local_values
+
+
+def get_wrapper_par(problem, target_key, allgather=False):
```

Comment:
This doesn't get rid of duplicate ghosts right? Probably worth mentioning.

Reply:
Great point.  Do you think that we should have an option to filter ghosts here, or should we add a note and leave it to the users?

Reply:
I think it would be a good idea to add that capability at some point, but it's a lot more complicated obviously.